mmv1/products/looker/Instance.yaml

```
@@ -0,0 +1,362 @@
+            description: |
+              Seconds of minutes of the time. Must normally be from 0 to 59.
+            validation: !ruby/object:Provider::Terraform::Validation
+              function: 'validation.IntBetween(0,60)'
+          - !ruby/object:Api::Type::Integer
+            name: 'nanos'
+            description: |
+              Fractions of seconds in nanoseconds. Must be from 0 to 999,999,999.
+            validation: !ruby/object:Provider::Terraform::Validation
+              function: 'validation.IntBetween(0,999999999)'
+  # Maintenance Window Object - End
+  - !ruby/object:Api::Type::String
+    name: name
+    description: |
+      The ID of the instance or a fully qualified identifier for the instance.
+    required: true
+    immutable: true
+    custom_expand: 'templates/terraform/custom_expand/shortname_to_url.go.erb'
+    custom_flatten: 'templates/terraform/custom_flatten/name_from_self_link.erb'
+    validation: !ruby/object:Provider::Terraform::Validation
+      regex: '^[a-z][a-z0-9-]{0,39}[a-z0-9]$'
+  # Oauth Object
+  - !ruby/object:Api::Type::NestedObject
+    name: oauthConfig
+    ignore_read: true
+    description: |
+      Looker Instance OAuth login settings.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: 'clientId'
+        description: |
+          The client ID for the Oauth config.
+      - !ruby/object:Api::Type::String
+        name: 'clientSecret'
+        description: |
+          The client secret for the Oauth config.
+  # Oauth Object - End
+  - !ruby/object:Api::Type::Enum
+    name: platformEdition
+    description: |
+      Platform editions for a Looker instance. Each edition maps to a set of instance features, like its size. Must be one of these values:
+      - LOOKER_CORE_TRIAL: trial instance
+      - LOOKER_CORE_STANDARD: pay as you go standard instance
+      - LOOKER_CORE_STANDARD_ANNUAL: subscription standard instance
+      - LOOKER_CORE_ENTERPRISE_ANNUAL: subscription enterprise instance
+      - LOOKER_CORE_EMBED_ANNUAL: subscription embed instance
+      - LOOKER_MODELER: standalone modeling service
+    values:
+      - :LOOKER_CORE_TRIAL
+      - :LOOKER_CORE_STANDARD
+      - :LOOKER_CORE_STANDARD_ANNUAL
+      - :LOOKER_CORE_ENTERPRISE_ANNUAL
+      - :LOOKER_CORE_EMBED_ANNUAL
+      - :LOOKER_MODELER
+    default_value: :LOOKER_CORE_TRIAL
+    immutable: true
+  - !ruby/object:Api::Type::Boolean
+    name: privateIpEnabled
+    description: |
+      Whether private IP is enabled on the Looker instance.
+    default_value: false
+  - !ruby/object:Api::Type::Boolean
+    name: publicIpEnabled
+    description: |
+      Whether public IP is enabled on the Looker instance.
+    default_value: true
+  - !ruby/object:Api::Type::String
+    name: reservedRange
+    description: |
+      Name of a reserved IP address range within the consumer network, to be used for
+      private servicie access connection. User may or may not specify this in a request.
```

```suggestion
      private service access connection. User may or may not specify this in a request.
```

This is a typo in the original API docs, fwiw

---

Fixed